docs/date-picker/datepicker.md

```diff
@@ -1,8 +1,122 @@
+
+## Elements
+* Input Component with Icon Component
+* Calendar View for selecting dates
+
+![Image of basic date picker](./assets/datepicker/DatePickerBasicInput.png)
+
+UI presentation of Calendar View behaviors:
+
+* *Nothing selected or focused*
+
+![Image of calendar view 1](./assets/datepicker/CalendarView-nothing.png)
+
+* *Same date selected and focused*
+
+![Image of calendar view 2](./assets/datepicker/CalendarView-SameSelectedFocused.png)
+
+* *Current day focused, not selected*
+
+![Image of calendar view 3](./assets/datepicker/CalendarView-SameSelectedCurrentFocus.png)
+
+* *Selected and focused on different days* - 18th selected, 20th focused
+
+![Image of calendar view 4](./assets/datepicker/CalendarView-SelectedFocusedDifferent.png)
+
+* *Selected, focused and current on different days* - 18th selected, 20th current day, 11th focused
+
+![Image of calendar view 5](./assets/datepicker/CalendarView-SelectedCurrentFocusedDifferent.png)
+
+* *Changing the first day of the week* - In this example the first day of the week is set to Monday.
+
+![Image of calendar view 6](./assets/datepicker/CalendarView-firstdayofweek.png)
+
+* *Show prev/next days* - the days are not clickable, just shown and considered disabled
+
+![Image of calendar view 7](./assets/datepicker/CalendarViewPrevNext.png)
+
+* *Disable Weekends* - the days are not clickable, just shown and considered disabled
+
+![Image of calendar view 8](./assets/datepicker/CalendarViewDisabledWeekends.png)
+
+* *Disable specific dates* - the days are not clickable, just shown and considered disabled
+
+![Image of calendar view 9](./assets/datepicker/CalendarView-DisableSpecificDates.png)
+
+* *Calendar View open - Bottom Left*
+
+![Image of calendar view 10](./assets/datepicker/CalendarView-OpenBL.png)
+
+* *Calendar View open - Bottom Right*
+
+![Image of calendar view 11](./assets/datepicker/CalendarView-OpenBR.png)
+
+* *Calendar View open - Top Left*
+
+![Image of calendar view 12](./assets/datepicker/CalendarView-OpenTL.png)
+
+* *Calendar View open - Top Right*
+
+![Image of calendar view 13](./assets/datepicker/CalendarView-OpenTR.png)
+
+## Properties
+
+DatePicker extends formInputProps
+
+| Name | Type | Default| Required | Description |
+| --- | --- | --- | --- | --- |
+| openOnFocus | boolean | false | no | Whether the Calendar View is opened when the input component gains focus. |
+| inputPlaceholder | string | empty string | no |Placeholder text for the input |
+| firstDayOfWeek | number | Defaults to 0 (Sunday), accepts 0 - 6 | no | define starting day of the week |
+| minDate | ISO 8601 date | empty date | no | Used to set the minimal date shown in the Calendar View and accepted in the input |
+| maxDate | ISO 8601 date | empty date | no | Used to set the maximum date shown in the Calendar View and accepted in the input |
+| inputFormat | string | browser implementation of javascript date object constructor | no | The format used to parse the input | 
```

Comment:
do we know how to validate this?

Reply:
not currently but we still don't implement it in general.